preload.py

```diff
@@ -0,0 +1,7 @@
+# preload.py is used for cmd line arguments
+def preload(parser):
+    parser.add_argument(
+        "--sqlite-file",
```

Comment:
I think it would be nicer if the argument has some prefix like `agent-scheduler-sqlite-file`.

Reply:
Made it to agent-scheduler-sqlite-file!